mlir/include/mlir/Dialect/Bufferization/Transforms/Passes.h

```diff
@@ -192,51 +165,20 @@ struct BufferResultsToOutParamsOpts {
   bool hoistStaticAllocs = false;
 };
 
-/// Creates a pass that converts memref function results to out-params.
-std::unique_ptr<Pass> createBufferResultsToOutParamsPass(
-    const BufferResultsToOutParamsOpts &options = {});
-
 /// Replace buffers that are returned from a function with an out parameter.
 /// Also update all call sites.
 LogicalResult
 promoteBufferResultsToOutParams(ModuleOp module,
                                 const BufferResultsToOutParamsOpts &options);
 
-/// Creates a pass that drops memref function results that are equivalent to a
-/// function argument.
-std::unique_ptr<Pass> createDropEquivalentBufferResultsPass();
-
-/// Create a pass that rewrites tensor.empty to bufferization.alloc_tensor.
-std::unique_ptr<Pass> createEmptyTensorToAllocTensorPass();
-
 /// Drop all memref function results that are equivalent to a function argument.
 LogicalResult dropEquivalentBufferResults(ModuleOp module);
 
-/// Create a pass that bufferizes all ops that implement BufferizableOpInterface
-/// with One-Shot Bufferize.
-std::unique_ptr<Pass> createOneShotBufferizePass();
-
-/// Create a pass that bufferizes all ops that implement BufferizableOpInterface
-/// with One-Shot Bufferize and the specified bufferization options.
-std::unique_ptr<Pass>
```

Comment:
Was it intentional that this was removed as well?
It doesn't look like this is generated with tablegen? Only a method that allows to pass OneShotBufferizePassOptions.

Those are different than OneShotBufferizationOptions:
https://github.com/llvm/llvm-project/blob/main/mlir/include/mlir/Dialect/Bufferization/Transforms/OneShotAnalysis.h